crates/dt-workspace/src/config.rs

```diff
@@ -0,0 +1,153 @@
+use serde::{Deserialize, Serialize};
+use std::{
+    io::ErrorKind,
+    path::{Path, PathBuf},
+};
+use thiserror::Error;
+
+use crate::search::search;
+
+pub const CONFIG_FILENAME: &'static str = ".dt-tools.toml";
+
+#[derive(Debug, PartialEq, Default, Serialize, Deserialize)]
+#[serde(rename_all = "kebab-case")]
+pub struct Config {
+    pub include_paths: Vec<PathBuf>,
+}
+
+impl Config {
+    /// Looks for [CONFIG_FILENAME] in `start` or cwd
```

Comment:
```suggestion
    /// Looks for [`CONFIG_FILENAME`] in `start` or cwd
```
Please use backticks around Rust identifiers